### PR TITLE
Added worst-case scenario section, link for rsync, etc.

### PR DESCRIPTION
**Pull request by storyaddict**

### CODE REVIEW

File: _articles/disaster-recovery.md [hunk 1]
```diff
@@ -46,7 +46,9 @@ Once the drive is mounted, we can install software to back up files.
 
 [Déjà Dup](https://wiki.gnome.org/Apps/DejaDup) is a popular, straightforward backup option. It can be used to back up user data locally, remotely, or to a cloud server. It can also be set up to run automatically on a regular basis.
 
-[rsync]() is a popular command-line tool to copy data while keeping the permissions. It is useful if the system is powering down or if you want a command-line tool to copy the files. 
+### Rsync
+
+[rsync](https://www.digitalocean.com/community/tutorials/how-to-use-rsync-to-sync-local-and-remote-directories) is a popular command-line tool to copy data while keeping the permissions. It is useful if the system is powering down or if you want a command-line tool to copy the files.
```
Comment (jaxsnjohnson) [inline _articles/disaster-recovery.md:51]: Please note the cloud storage policy may affect upload/download speed.  This was an issue at an old Job @zachwingrave and I had, the uploads from Linux were snail-paced.

File: _articles/disaster-recovery.md [hunk 2]
```diff
@@ -100,11 +102,16 @@ Now that we reinstalled the OS we can start restoring our data. To restore your
...
 ![Restoring data in Deja-Dup](/images/backup/deja_overview.png)
 
-## If the system doesn't power on
+## If the System Doesn't Power On
 
 Refer to your models [service manual](/articles/service-manuals) for removing the drive(s). Once you have moved the drives containing your data to another system, you can transfer them to an external hard drive (or other location) with the steps in our [backup article](/articles/backup-files).
 
-## Future Steps
+## What if the Drive is the Problem?
+If signs are pointing to the drive itself being the issue, and your files are corrupted, or no longer accessible, a data recovery service may be the only viable option. If that is the case, remove the drive, and stop using it until it can be taken to a data recovery site. Continued use of the drive may exacerbate bad sectors, or overwrite existing, partially retrievable data.
```
Comment (jaxsnjohnson) [inline _articles/disaster-recovery.md:110]: Do we want to link to any good ones?

**Review by jaxsnjohnson — CHANGES_REQUESTED** [review]

File: _articles/disaster-recovery.md (hunk 2 above)
Comment (storyaddict) [reply _articles/disaster-recovery.md:110]: Don't want to open that can of worms.

File: _articles/disaster-recovery.md (hunk 1 above)
Comment (storyaddict) [reply _articles/disaster-recovery.md:51]: Added a note related to this.

**Comment by jaxsnjohnson** [comment]
Looks good, nice changes.

**Review by jaxsnjohnson — APPROVED** [review]
Good Changes, and additions to the article.

**Pull request merged**